skyreader/plot/plotting_tools.py

```diff
@@ -259,4 +259,4 @@ def _get_affine_transform(self):
         _, yscale = transform.transform_point((0, np.pi / 2.0))
         return Affine2D() \
             .scale(0.5 / xscale, 0.5 / yscale) \
-            .translate(0.5, 0.5)
\ No newline at end of file
+            .translate(0.5, 0.5)
```

Comment:
I don't like the `\`-syntax to split between lines. If it doesn't all fit on one line, I recommend taking a look at how the black formatter does this: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#call-chains